src/server/api/lib/twilio.js

```diff
@@ -10,6 +10,8 @@ import {
 import { log } from "../../../lib";
 import { saveNewIncomingMessage } from "./message-sending";
 import { getConfig } from "./config";
+import _ from "lodash";
+import urlJoin from "url-join";
```

Comment:
doesn't look like you're using this method or library -- can we delete it after all?